addons/mail/static/tests/helpers/model_definitions_setup.js

```diff
@@ -80,6 +80,10 @@ insertModelFields('mail.tracking.value', {
 });
 insertModelFields('res.partner', {
     description: { string: 'description', type: 'text' },
+    user_ids: { string: "Users", type: "one2many", relation: 'res.users', default:[] },
+});
+insertModelFields('res.users', {
+    share: { string: "Shared users", type: 'boolean', default: false },
```

Comment:
Same question, is it really necessary? Maybe just to define the default value in this case? But keeping default undefined on a boolean probably works when its indented to be false.